workflow/scripts/postprocess_diffexp.py

```
@@ -35,8 +39,12 @@ def sort_rows(df):
     columns_with_prefix = [col for col in df.columns if col.startswith(signed_pi_start)]
 
     if len(columns_with_prefix) != 1:
-        raise ValueError(
-            f"Expected exactly one column starting with '{signed_pi_start}', found {len(columns_with_prefix)}"
+        warning.warn(
```

Ruff (0.8.2)

Undefined name `warning`

(F821)